templates/templates/navigation-enterprise-h.html

```diff
@@ -82,7 +82,7 @@ <h4 class="u-show--small p-muted-heading u-hide--medium u-hide--large">
             <li class="p-list__item"><a href="/appliance">Appliance images</a></li>
             <li class="p-list__item"><a href="/embedded">Embedded Linux with Ubuntu</a></li>
             <li class="p-list__item"><a href="https://snapcraft.io" title="Visit Snapcraft - external site">Snaps - secure packages for IoT</a></li>
-            <li class="p-list__item"><a href="/download/iot">Supported boards and SoCs</a></li>
+            <li class="p-list__item"><a href="/certification">Supported boards and SoCs</a></li>
```

Comment:
Shouldn't this point to `certification?form=Ubuntu%20Core`?

Reply:
This is in the wireframe: "In the “Enterprise” > Internet of Things” mega-menu section, the “Supported boards and SoCs” item should link to the main page, since it refers to two form factors"